docs/debug.md

```diff
@@ -0,0 +1,48 @@
+# CMake Debugging
+
+Starting with CMake 3.27, debugging CMake is now supported in CMake Tools!
+
+The following documentation will help you understand the various ways you can debug CMake.
+
+## Debugging from CMake Tools UI entry points
+
+The most common reason to debug CMake is to debug CMake Configuration! There are many ways that you can accomplish this:
+
+* Commands
+  * CMake: Configure with CMake Debugger
+  * CMake: Delete Cache and Reconfigure with CMake Debugger
+* Folder Explorer
+  * Right click on CMakeLists.txt context menu entry.
```

Comment:
"context menu entry" feel redundant right after "right click". Can we instead tell the user what to actually click on?

Reply:
Modifying.